app/src/main/java/org/eyeseetea/malariacare/data/database/model/Survey.java

```diff
@@ -311,6 +314,21 @@ public void setEventUid(String eventuid) {
     }
 
     /**
+     * Ratio of completion is cached into answeredQuestionRatio in order to speed up loading
+     */
+    public SurveyAnsweredRatio getAnsweredQuestionRatio() {
```

Comment:
@manuelplazaspalacio SurveyAnsweredRatio is a domain concept not a data concept than cache management is better that be realized by new use case and not by database model